mobile/src/bundles/talent/store/actions.ts

```diff
@@ -58,13 +57,16 @@ const updateOnboardingData = createAsyncThunk<
 });
 
 const getTalentDetails = createAsyncThunk<
-    UserDetailsResponseDto | null,
-    UserDetailsFindRequestDto,
+    UserDetailsGeneralResponseDto | null,
+    UserDetailsGeneralResponseDto,
     AsyncThunkConfig
->(`${sliceName}/getTalentDetails`, (detailsPayload, { extra }) => {
-    const { talentApi, notifications } = extra;
+>(`${sliceName}/getTalentDetails`, async (payload, { extra }) => {
```

Comment:
this endpoint will be used both for talent and employer

Reply:
If you mean getUserDetailsById yes, we will use it for employer too, but do we need create new class only for this request?

Reply:
It shouldn't be named `getTalentDetails`

Reply:
we'll use this action for both roles